src/fsharp/ParseHelpers.fs

```diff
@@ -181,3 +184,51 @@ type LexerWhitespaceContinuation =
 
 and LexCont = LexerWhitespaceContinuation
 
+//------------------------------------------------------------------------
```

Comment:
This is wierd, everywhere else there were ------------ you removed them, yet here you didn't.

Reply:
Yes they should go there too.